src/go/rpk/pkg/cli/group/group.go

```diff
@@ -91,6 +97,13 @@ This command lists all groups currently known to Redpanda, including empty
 groups that have not yet expired. The BROKER column is which broker node is the
 coordinator for the group. This command can be used to track down unknown
 groups, or to list groups that need to be cleaned up.
+
+The STATE columns shows which state the group is in:
+  - PreparingRebalance: The group is preparing to rebalance.
+  - CompletingRebalance: The group is waiting on the leader to provide assignments.
+  - Stable: The group is stable.
```

Comment:
Does "Stable" mean all members are "alive" and could that also mean there might be some that are idle? Or does stable mean, no changes/scaling in progress?

Reply:
> Does "Stable" mean all members are "alive" and could that also mean there might be some that are idle? Or does stable mean, no changes/scaling in progress?
The latter.

I've [expanded](https://github.com/redpanda-data/redpanda/compare/b941f4f11441620d2d691378aaea383a0a9a97a1..569a3eaac8e26add2494d9a6a22ad39997c0613c) this line to `Stable: The group is not empty and has no group membership changes in process.`. Let me know if that makes sense.